hardhat.config.ts

```diff
@@ -29,6 +29,15 @@ const config: HardhatUserConfig = {
       gasPrice: 0,
     },
   },
+  solidity: {
+    version: '0.8.18',
+    settings: {
+      optimizer: {
+        enabled: true,
+        runs: 200,
+      },
+    },
```

Comment:
id did add viaIR: true, in my. pr depending on who merges first we need to check